lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -899,6 +885,29 @@ public function get_item_schema() {
 				),
 			),
 		);
+
+		if ( '1' === get_option( 'show_avatars' ) ) {
```

Comment:
I think this should be a `if ( get_option...` the strict type check, while correct for most cases, is somewhat unintuitive, as this _really_ should be a bool. +1 for allowing any truthy value.